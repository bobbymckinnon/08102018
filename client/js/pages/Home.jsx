import React from 'react'
import { Div } from 'glamorous'

class Home extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <Div className="container">
                <Div className="col-xs-8 col-xs-offset-2 jumbotron text-center">
                    <h1>Patient Data</h1>
                    <p>You have been asked to assist in the creation of an internal API & Search page for a national healthcare provider. This provider has a set of inpatient prospective payment systems providers.</p>
                    <a onClick={()=>this.props.showFilter(true)} className="btn btn-primary btn-lg btn-login btn-block">Search Patient Data</a>
                </Div>
            </Div>
        )
    }
}

export default Home
