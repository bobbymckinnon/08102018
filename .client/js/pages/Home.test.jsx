import React from 'react';
import { shallow } from 'enzyme';

import Home from '../../../client/js/pages/Home';

describe('Home', () => {
    it('should render Home correctly', () => {
        const component = shallow(<Home />);
        expect(component).toMatchSnapshot();
    });

    it("always renders a link to the search page", () => {
        const component = shallow(<Home />);
        const a = component.find("a");
        expect(a.length).toBeGreaterThan(0);
    });
});
