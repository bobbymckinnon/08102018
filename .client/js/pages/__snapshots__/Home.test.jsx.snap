// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home correctly 1`] = `
<glamorous.Div
  className="container"
>
  <glamorous.Div
    className="col-xs-8 col-xs-offset-2 jumbotron text-center"
  >
    <h1>
      Patient Data
    </h1>
    <p>
      You have been asked to assist in the creation of an internal API & Search page for a national healthcare provider. This provider has a set of inpatient prospective payment systems providers.
    </p>
    <a
      className="btn btn-primary btn-lg btn-login btn-block"
      onClick={[Function]}
    >
      Search Patient Data
    </a>
  </glamorous.Div>
</glamorous.Div>
`;
